src/audio/host-zephyr.c

```diff
@@ -1075,7 +981,7 @@ static int host_reset(struct comp_dev *dev)
 	hd->copy_type = COMP_COPY_NORMAL;
 	hd->source = NULL;
 	hd->sink = NULL;
-	dev->state = COMP_STATE_READY;
+	hd->state = COMP_STATE_READY;
```

Comment:
@btian1 please dont move the state. We need to use dev->state. But instead of chaning the state to host_data struct, set the state for the host dev in copier_prepare

Reply:
ok, then we have to use: cd->endpoint[i] as host device, this should be allowed, right?